pkg/combustion/templates/12-suma-register.sh.tpl

```diff
@@ -21,4 +21,4 @@ enable_fqdns_grains: False
 
 EOF
 
-systemctl restart venv-salt-minion || true
+systemctl enable venv-salt-minion || true
```

Comment:
Do we need `|| true` now?

Reply:
@Kristian-ZH arguably not... just playing it safe with this as we don't really want the installation to completely fail if the venv-salt-minion can't start.